docs/integrations/context-standards-mandatory.md

```diff
@@ -1,22 +1,12 @@
 ---
-id: context-standards
-title: Context Standards
+id: context-standards-mandatory
+title: Mandatory Context Standards
 ---
 
-Cortex XSOAR organizes incident data in a tree of objects called the *Incident Context*. Any integration commands or scripts that run, will add data into the context at a predefined location. This also applies to commands that run within playbook execution.
-
-When building new integrations the entry context should be returned according to this standard. 
-
-If there is no matching item in the standard, reach out to the architect team to check if your addition merits an update or change of the standard. 
-
-If the data is vendor/tool specific, it can be assigned its own context path.
-
-Below are examples of how each entity should be formed in the entry context.
-
 ## File
 The following is the format for a file.
```

Comment:
Worth mentioning what we mean by `File` and the differences from `InfoFile` below.